.github/workflows/ci.yml

```diff
@@ -1,17 +1,14 @@
 name: CI
 on: [push]
-
+env:
```

Comment:
fun fact! you can just specify an `env:` at the top-level and that environment var is applied to all jobs.

Reply:
That's awesome.